### ISSUE TITLE
Accessibility contrast checker should fail if it only finds a single color

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#32058]

**Comment by jonahwilliams** [comment]
This is already done - we use the same color for the CR and get a ratio of 1
